coredocument/coredocument.proto

```diff
@@ -100,8 +100,10 @@ message AccessToken {
   bytes grantee = 4;
   // Role identifier is the identifier on the read rule that this token should be mapped to
   bytes role_identifier = 5;
-  // Original identifier of the document
+  // Identifier of the document where the access token can be found
   bytes document_identifier = 2;
+  // Identifier of the delegate document which can be accessed via the access token
+  bytes delegated_document_identifier = 8;
```

Comment:
Unless you are going to reuse the same message when requesting access to a document, I believe this field here is not needed, as the AccessToken lives implicitly under the document identified by the `delegated_document_identifier `. And is responsibility of the requester at the moment, to tell the receiver where to find the token.

Reply:
I think the naming here is confusing then, maybe we should change document_identifier to delegated document identifier, as it is unclear from just document_identifier which document the identifier is referring to. In the validation process of the AT, this is important because we need to verify that the granter has access to both the delegating and delegated document.

Reply:
shelving this post discussion